.github/workflows/ci.yaml

```diff
@@ -6,28 +6,89 @@ on:
 
 jobs:
   tests:
-    runs-on: ubuntu-20.04
+    runs-on: ${{matrix.os}}
     name: Test (${{matrix.elixir}}/${{matrix.otp}})
 
     strategy:
+      fail-fast: false
       matrix:
-        otp: [23.x, 24.x, 25.x]
-        elixir: [1.13.x, 1.14.x]
+        include:
```

Comment:
This seems a bit overkill in my opinion. 

Testing in ubuntu-latest should be fine as well as just testing the last three elixir versions and OTP versions.

Reply:
I do it a bit simpler in [app_identity](https://github.com/KineticCafe/app_identity) and can simplify this, but my *recommendation* would be to instead test on each *supported* version with the maximum supported OTP version on the best supported OS for that (which is not always going to be `ubuntu-latest`).

This will be affected by the possible minimum version discussion in #94 as well.

Reply:
The standard matrix should be fine for this package, and ubuntu-latest should be okay, the only reason it had ubuntu-20.04 before was because OTP-23 wasn't available on newer distributions.

I am satisfied with testing elixir 1.17, 1.16, and 1.15 and OTP25-27 (only 1.17 is compatible with 27 tho)

The minimum elixir version can probably be bumped to 1.15. That is the version of Elixir installed in the Netlify build images (and how I deploy my sites)